lexos/processors/analyze/topword.py

```diff
@@ -1,580 +1,248 @@
-                topword analysis
-        low: the raw count lower bound of the words that send into the
-                topword analysis
-    :raise IOError: the option you put in is not recognized by the program
+def _z_test_word_list_(count_list_i: np.ndarray, count_list_j: np.ndarray,
+                       words: np.ndarray) -> List[Tuple[str, int]]:
+    """Processes z-test on all the words of two input word lists.
 
+    :param count_list_i: first word list, a numpy arrant contains word counts.
+    :param count_list_j: second word list, a numpy arrant contains word counts.
+    :param words: words that show up at least one time in the whole corpus.
+    :return: a list that is sorted by z-scores contains tuples, where each type
+             maps a word to its z-score.
     """
-    if option == 'CustomP':
-        low *= total_word_count
-        high *= total_word_count
-
-    elif option == 'CustomR':  # custom raw counts
-        pass
-
-    elif option.endswith('std_err'):
-        std_err = 0
-        # average frequency of the word appearance (raw count)
-        average = total_word_count / num_word
-        for word in merge_list:
-            std_err += (merge_list[word] - average) ** 2
-        std_err = sqrt(std_err)
-        std_err /= num_word
-
-        if option.startswith('top'):
-            # TopStdE: only analyze the Right outlier of word, determined by
-            # standard deviation
-            high = total_word_count
-            low = average + 2 * std_err
-
-        elif option.startswith('mid'):
-            # MidStdE: only analyze the Non-Outlier of word, determined by
-            # standard deviation
-            high = average + 2 * std_err
-            low = average - 2 * std_err
-
-        elif option.startswith('low'):
-            # LowStdE: only analyze the Left Outlier of word, determined by
-            # standard deviation
-            high = average - 2 * std_err
-
-        else:
-            raise IOError('input option is not valid')
-
-    elif option.endswith('iqr'):
-        temp_list = sorted(list(merge_list.items()), key=itemgetter(1))
-        mid = temp_list[int(num_word / 2)][1]
-        q3 = temp_list[int(num_word * 3 / 4)][1]
-        q1 = temp_list[int(num_word / 4)][1]
-        iqr = q3 - q1
-
-        if option.startswith('top'):
-            # TopIQR: only analyze the Top outlier of word, determined by iqr
-            high = total_word_count
-            low = (mid + 1.5 * iqr)
-
-        elif option.startswith('mid'):
-            # MidIQR: only analyze the non-outlier of word, determined by iqr
-            high = (mid + 1.5 * iqr)
-            low = (mid - 1.5 * iqr)
-
-        elif option.startswith('low'):
-            # LowIQR: only analyze the Left outlier of word, determined by iqr
-            high = (mid - 1.5 * iqr)
-
-        else:
-            raise IOError('input option is not valid')
+    # initialize
+    word_z_score_dict = {}
```

Comment:
Sorry I may need a bit more time to change is.. Sorry about keeping this PR this long